src/CommunityToolkit.Maui/Behaviors/ICommunityToolkitBehavior.shared.cs

```diff
@@ -26,11 +26,13 @@ internal bool TrySetBindingContextToAttachedViewBindingContext()
 			return false;
 		}
 
+#pragma warning disable IL2026 // NET9.0 - check if this can be fixed
```

Comment:
I have suppressed this for now but we should consider whether we can avoid it.

TypedBinding didn't work here due to the accessibility of `View`

Reply:
I've resolved the builder errors for now by adding the `[RequiresUnreferencedCode]` attribute to `ValidationBehavior`. 

After we release .NET 9 support, we can circle back to this to see if there is a way to avoid using string bindings with `ValidationBehavior`.